src/cdk/tree/tree.spec.ts

```diff
@@ -595,6 +595,21 @@ describe('CdkTree', () => {
             [_, _, `topping_6 - cheese_6 + base_6`],
             [`topping_3 - cheese_3 + base_3`]);
       });
+
+      it('with correct aria-level', () => {
```

Comment:
I know the tests were like this already, but could you phrase this as a sentence? E.g.
```ts
it('should set the correct aria-level on nodes', () => {
```

Reply:
The way the tests are set up will cause the test message to become 
"CdkTree nested tree should initialize should set the correct aria-level on nodes"